### PR TITLE
Bump app_profiler to 0.1.1

### PR DESCRIPTION
**Pull request by dalehamel**
Created with:

- `gem bump`
- bundle
- git commit --amend (to add changelog and gemfile lock to bump commit)

### CODE REVIEW

**Comment by dalehamel** [comment]
PTAL @gmcgibbon ❤️

**Review by bmansoob — APPROVED** [review]
LGTM -- If I am remembering the workflow correctly, adding a new release( https://github.com/Shopify/app_profiler/releases) happens after merge.

**Review by rafaelfranca — APPROVED** [review]

**Pull request merged**
